generated-docs/out/backends/summary.md

```diff
@@ -19,29 +19,30 @@ Each backend has two type parameters:
-``AsyncHttpClientMonixBackend``      ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  no
-``AsyncHttpClientCatsBackend``       ``F[_]: cats.effect.Concurrent`` n/a                                               no                         no
-``AsyncHttpClientFs2Backend``        ``F[_]: cats.effect.Concurrent`` ``fs2.Stream[F, Byte]``                           yes (regular & streaming)  no
-``OkHttpSyncBackend``                None (``Identity``)              n/a                                               yes (regular)              no
-``OkHttpFutureBackend``              ``scala.concurrent.Future``      n/a                                               yes (regular)              no
-``OkHttpMonixBackend``               ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  no
-``Http4sBackend``                    ``F[_]: cats.effect.Effect``     ``fs2.Stream[F, Byte]``                           no                         no
-``HttpClientSyncBackend``            None (``Identity``)              n/a                                               no                         no
-``HttpClientFutureBackend``          ``scala.concurrent.Future``      n/a                                               yes (regular)              no
-``HttpClientMonixBackend``           ``monix.eval.Task``              ``monix.reactive.Observable[ByteBuffer]``         yes (regular & streaming)  yes
-``HttpClientFs2Backend``             ``F[_]: cats.effect.Concurrent`` ``fs2.Stream[F, Byte]``                           yes (regular & streaming)  yes
-``HttpClientZioBackend``             ``zio.Task``                     ``zio.stream.Stream[Throwable, Byte]``            yes (regular & streaming)  yes
-``FinagleBackend``                   ``com.twitter.util.Future``      n/a                                               no                         no
-==================================== ================================ ================================================= ========================== ===================
+==================================== ================================ ================================================= ========================== =================== ===============================
+Class                                Effect type                      Supported stream type                             Supports websockets        Fully non-blocking  Supports host header overrides 
+==================================== ================================ ================================================= ========================== =================== ===============================
+``HttpURLConnectionBackend``         None (``Identity``)              n/a                                               no                         no                  no
```

Comment:
also for the built-in-java backends, a note on how to override this behavior would be good (there's some system property), so that the documentation is helpful

Reply:
I Added note about system property for HttpURLConnection and HttpClient backend sections